docs/root/adaptive_load_controller.md

```diff
@@ -0,0 +1,202 @@
+Minimum latency         | `builtin`   | `latency-ns-min`              | The minimum latency in nanoseconds observed by Nighthawk in the iteration.
+Mean latency            | `builtin`   | `latency-ns-mean`             | The average latency in nanoseconds observed by Nighthawk in the iteration.
+Maximum latency         | `builtin`   | `latency-ns-max`              | The maximum latency in nanoseconds observed by Nighthawk in the iteration.
+Mean + 1x stdev latency | `builtin`   | `latency-ns-mean-plus-1stdev` | Calculated as average latency plus one standard deviation in nanoseconds as observed by Nighthawk in the iteration.
+Mean + 2x stdev latency | `builtin`   | `latency-ns-mean-plus-2stdev` | Calculated as average latency plus two standard deviations in nanoseconds as observed by Nighthawk in the iteration.
+Mean + 3x stdev latency | `builtin`   | `latency-ns-mean-plus-3stdev` | Calculated as average latency plus three standard deviations in nanoseconds as observed by Nighthawk in the iteration.
+Stdev latency           | `builtin`   | `latency-ns-pstdev`           | The standard deviation of latencies in nanoseconds as observed by Nighthawk in the iteration.
+
+### Available step controllers
+
+The existing step controllers are implemented in the
+[step_controller_impl.cc](https://github.com/envoyproxy/nighthawk/blob/main/source/adaptive_load/step_controller_impl.cc)
+file.
+
+Step controller      | Plugin Name                    | Description
+-------------------- | ------------------------------ | -----------
+Exponential search   | `nighthawk.exponential_search` | Implements the exponential search algorithm.
+
+### Available metric scoring functions
+
+The existing metric scoring functions are implemented in the
+[scoring_function_impl.cc](https://github.com/envoyproxy/nighthawk/blob/main/source/adaptive_load/scoring_function_impl.cc)
+file.
+
+Scoring function | Plugin Name                | Description
+---------------- | -------------------------- | -----------
+Binary scoring   | `nighthawk.binary_scoring` | A boolean (true/false) evaluation of the metric value.
+Linear scoring   | `nighthawk.linear_scoring` | Evaluation of the metric value with a linear scoring constant.
+
+### Available input variable setters
+
+The existing input variable setters are implemented in the
+[input_variable_setter_impl.cc](https://github.com/envoyproxy/nighthawk/blob/main/source/adaptive_load/input_variable_setter_impl.cc)
+file.
+
+Input variable setter | Plugin Name     | Description
+--------------------- | --------------- | -----------
+RPS                   | `nighthawk.rps` | Sets the RPS value in the Nighthawk configuration for an iteration.
+
+## Configuration
+
+The controller accepts its
+[configuration](https://github.com/envoyproxy/nighthawk/blob/main/api/adaptive_load/adaptive_load.proto)
+in protocol buffer format.
+
+A valid configuration contains a `nighthawk_traffic_template` which is the base
```

Comment:
I believe there are other substitutions beyond the variable setter that we might want to reiterate here.  You already referenced above that even if the template indicates closed loop, it will be overwritten to open loop here?

Reply:
Added a note about the open loop setting always being overridden and also included a link to code where other defaults are set.

Done in b9326c26dc7825b4111831ecc0cc5571de12ca6a.